packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt

```diff
@@ -34,6 +35,12 @@ data class GutenbergProps @JvmOverloads constructor(
         }
     }
 
+    fun getUpdatedCapabilitiesProps() = WritableNativeMap().apply {
+        putBoolean(PROP_CAPABILITIES_MENTIONS, enableMentions)
+        putBoolean(PROP_CAPABILITIES_UNSUPPORTED_BLOCK_EDITOR, enableUnsupportedBlockEditor)
+        putBoolean(PROP_CAPABILITIES_MODAL_LAYOUT_PICKER, isModalLayoutPickerEnabled)
+    }
```

Comment:
I think it would be good to reuse this logic when we [create the capabilities for the initial props](https://github.com/WordPress/gutenberg/blob/d92ebe1a60ca940c0250df0cd223a141ef0481b3/packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt#L27-L29). I know that we need a `Bundle` in the initial props and we're returning a `WritableNativeMap` here, but what about having this function return a `Bundle`, and then when we are "updating" the capabilities (and need a `WritableNativeMap`) we could still use this method, we would just need to convert the `Bundle` to a `WritableNativeMap`. Fortunately, that's pretty easy to do because React Native has a built-in helper method to do that: https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/Arguments.java#L136-L146.

This is my preference, but I don't think it's a blocker for the PR.

Reply:
Nice, sounds good to me! Thanks!

Reply:
@mchowning this is fixed, could you take a second look at the fix? thanks.

Reply:
I like it! Thanks!